app.js

```diff
@@ -62,35 +94,125 @@ function startGame() {
 }
 
 function Step() {
-  renderEnemy(enemy1);
-  renderEnemy(enemy2);
-  renderEnemy(enemy3);
+  renderEnemy(enemyHealth50text, healthBar50text, enemyHealth50, healthBar50, enemy1);
+  renderEnemy(enemyHealth100text, healthBar100text, enemyHealth100, healthBar100, enemy2);
+  renderEnemy(enemyHealth200text, healthBar200text, enemyHealth200, healthBar200, enemy3);
   renderBullet (bullet);
   enemy1.x = moveEnemy(enemy1.x, enemy1Velocity);
   enemy2.x = moveEnemy(enemy2.x, enemy2Velocity);
   enemy3.x = moveEnemy(enemy3.x, enemy3Velocity);
+  enemyHealth50.x = moveEnemy(enemyHealth50.x, enemyHealth50Velocity);
+  enemyHealth100.x = moveEnemy(enemyHealth100.x, enemyHealth100Velocity);
+  enemyHealth200.x = moveEnemy(enemyHealth200.x, enemyHealth200Velocity);
+  enemyHealth50text.x = moveEnemy(enemyHealth50text.x, enemyHealth50textVelocity);
+  enemyHealth100text.x = moveEnemy(enemyHealth100text.x, enemyHealth100textVelocity);
+  enemyHealth200text.x = moveEnemy(enemyHealth200text.x, enemyHealth200textVelocity);
+  gutling.x = moveEnemy(gutling.x, gutlingVelocity);
   bullet.x += bullet.velocityX;
   bullet.y += bullet.velocityY;
   moveRocket();
   checkEnemyShipCollision(enemy1);
   checkEnemyShipCollision(enemy2);
   checkEnemyShipCollision(enemy3);
+  ammo.innerHTML = `Gutling Ammo: ${gutling.ammo} <br> Rocket Ammo: ${rocket.ammo}`;
   if (checkEnemyRocketCollision(enemy1.x, enemy1.y)) {
-    enemy1.y += 800;
+    healthBar50.value -= 10;
+    rocket.dmg -= 10;
+    if (healthBar50.value <= 0) {
+      enemy1.x -= 9999;
+    }
+    if (rocket.dmg <= 0) {
+      rocket.x = airfighter.x +5;
+      rocket.y = airfighter.y +67;
+      rocketVelocity -= 8;
+      rocket.dmg = 50;
+      rocket.element.src = "Rocket.gif";
+    }
   }
   if (checkEnemyRocketCollision(enemy2.x, enemy2.y)) {
-    enemy2.y += 800;
+    healthBar100.value -= 10;
+    rocket.dmg -= 10;
+    if (healthBar100.value <= 0) {
+      enemy2.x -= 9999;
+    }
+    if (rocket.dmg <= 0) {
+      rocket.x = airfighter.x +5;
+      rocket.y = airfighter.y +67;
+      rocketVelocity -= 8;
+      rocket.dmg = 50;
+      rocket.element.src = "Rocket.gif";
+    }
   }
   if (checkEnemyRocketCollision(enemy3.x, enemy3.y)) {
-    enemy3.y += 800;
+    healthBar200.value -= 10;
+    rocket.dmg -= 10;
+    if (healthBar200.value <= 0) {
+      enemy3.x -= 9999;
+    }
+    if (rocket.dmg <= 0) {
+      rocket.x = airfighter.x +5;
+      rocket.y = airfighter.y +67;
+      rocketVelocity -= 8;
+      rocket.dmg = 50;
+      rocket.element.src = "Rocket.gif";
+    }
+  }
+  if (rocket.x > airfighter.x + 1500) {
+    rocket.x = airfighter.x +5;
+    rocket.y = airfighter.y +67;
```

Comment:
Number 67 is repeated on [line 140](https://github.com/DarthVanger/ghost-of-kyiv/pull/5/files#diff-e07d531ac040ce3f40e0ce632ac2a059d7cd60f20e61f78268ac3be015b3b28fR140).
If you create a variable `const something = 67`, you can use it multiple times, without copy-pasting the number 67/